cmd/helm-sync/main.go

```diff
@@ -37,10 +37,10 @@ var (
 		"the name of the helm chart being synced")
 	flVersion = flag.String("version", os.Getenv(reconcilermanager.HelmChartVersion),
 		"the version of the helm chart being synced")
-	flValues = flag.String("values", os.Getenv(reconcilermanager.HelmValues),
-		"set the helm chart values, will be used to override the default values")
-	flValuesFileSources = flag.String("values-file-sources", os.Getenv(reconcilermanager.HelmValuesFileSources),
-		"set the helm chart values, will be used to override the default values")
+	flValuesInline = flag.String("inline-values", os.Getenv(reconcilermanager.HelmValuesInline),
```

Comment:
yaml formatted cli flag value is pretty silly... but i guess it predates the new values files, so whatever...